writer/util.go

```diff
@@ -1,12 +1,20 @@
 package writer
 
-import "embed"
+import (
+	"embed"
+	"io/fs"
+	"regexp"
+)
 
 // Embed the templates directory in the compiled binary.
 //
 //go:embed templates
 var templates embed.FS
 
+var getFS = func() fs.FS {
```

Comment:
Any reason for declaring getFS as variable?

Reply:
I'm unable to replace the `templates` var directly in test cases due to this error
`cannot use fstest.MapFS{…} (value of type fstest.MapFS) as embed.FS value in assignment`

So, This is the one option that I could find, to convert the embed.FS to FS interface